cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -79,7 +79,7 @@ set(ARROW_THIRDPARTY_DEPENDENCIES
 # supported when building with MSVC because of the way that
 # conda-forge packages have 4 variants of the libraries packaged
 # together
-if(MSVC AND "${GTest_SOURCE}" STREQUAL "")
+if(MSVC AND "${GTest_SOURCE}" STREQUAL "" AND "${ARROW_DEPENDENCY_SOURCE}" STREQUAL "CONDA")
```

Comment:
Could you show full build log on error?
I think that avoiding old header and new library mix is the real fix.

Reply:
Here's the build error:

```
[build] [2/174] Linking CXX shared library debug\arrow_testing.dll
[build] FAILED: debug/arrow_testing.dll debug/arrow_testing.lib 
[build] cmd.exe /C "cd . && "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_dll --intdir=src\arrow\CMakeFiles\arrow_testing_shared.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1430~1.307\bin\Hostx64\x64\link.exe /nologo src\arrow\CMakeFiles\arrow_testing_shared.dir\io\test_common.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\ipc\test_common.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\json_integration.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\json_internal.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\gtest_util.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\random.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\generator.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\util.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\csv\test_common.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\compute\exec\test_util.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\filesystem\test_util.cc.obj  /out:debug\arrow_testing.dll /implib:debug\arrow_testing.lib /pdb:debug\arrow_testing.pdb /dll /version:700.0 /machine:x64  /NODEFAULTLIB:LIBCMT /debug /INCREMENTAL  debug\arrow.lib  googletest_ep-prefix\lib\gtestd.lib  ..\..\vcpkg_installed\x64-windows\lib\brotlienc.lib  ..\..\vcpkg_installed\x64-windows\lib\brotlidec.lib  ..\..\vcpkg_installed\x64-windows\lib\brotlicommon.lib  ..\..\vcpkg_installed\x64-windows\debug\lib\utf8proc.lib  ..\..\vcpkg_installed\x64-windows\debug\lib\re2.lib  mimalloc_ep\src\mimalloc_ep\lib\mimalloc-1.7\mimalloc-static-debug.lib  ws2_32.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib  && cmd.exe /C "cd /D C:\Users\voltron\arrow\cpp\build\user-python-debug-windows\src\arrow && C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -noprofile -executionpolicy Bypass -file C:/Users/voltron/arrow/vcpkg/scripts/buildsystems/msbuild/applocal.ps1 -targetBinary C:/Users/voltron/arrow/cpp/build/user-python-debug-windows/debug/arrow_testing.dll -installedDir C:/Users/voltron/arrow/cpp/vcpkg_installed/x64-windows/debug/bin -OutVariable out""
[build] LINK Pass 1: command "C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1430~1.307\bin\Hostx64\x64\link.exe /nologo src\arrow\CMakeFiles\arrow_testing_shared.dir\io\test_common.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\ipc\test_common.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\json_integration.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\json_internal.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\gtest_util.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\random.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\generator.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\testing\util.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\csv\test_common.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\compute\exec\test_util.cc.obj src\arrow\CMakeFiles\arrow_testing_shared.dir\filesystem\test_util.cc.obj /out:debug\arrow_testing.dll /implib:debug\arrow_testing.lib /pdb:debug\arrow_testing.pdb /dll /version:700.0 /machine:x64 /NODEFAULTLIB:LIBCMT /debug /INCREMENTAL debug\arrow.lib googletest_ep-prefix\lib\gtestd.lib ..\..\vcpkg_installed\x64-windows\lib\brotlienc.lib ..\..\vcpkg_installed\x64-windows\lib\brotlidec.lib ..\..\vcpkg_installed\x64-windows\lib\brotlicommon.lib ..\..\vcpkg_installed\x64-windows\debug\lib\utf8proc.lib ..\..\vcpkg_installed\x64-windows\debug\lib\re2.lib mimalloc_ep\src\mimalloc_ep\lib\mimalloc-1.7\mimalloc-static-debug.lib ws2_32.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:src\arrow\CMakeFiles\arrow_testing_shared.dir/intermediate.manifest src\arrow\CMakeFiles\arrow_testing_shared.dir/manifest.res" failed (exit code 1120) with the following output:
[build]    Creating library debug\arrow_testing.lib and object debug\arrow_testing.exp
[build] gtest_util.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class testing::AssertionResult __cdecl testing::internal::CmpHelperEQ(char const *,char const *,__int64,__int64)" (__imp_?CmpHelperEQ@internal@testing@@YA?AVAssertionResult@2@PEBD0_J1@Z) referenced in function "public: static class testing::AssertionResult __cdecl testing::internal::EqHelper::Compare(char const *,char const *,__int64,__int64)" (?Compare@EqHelper@internal@testing@@SA?AVAssertionResult@3@PEBD0_J1@Z)
[build] test_util.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class testing::AssertionResult __cdecl testing::internal::CmpHelperEQ(char const *,char const *,__int64,__int64)" (__imp_?CmpHelperEQ@internal@testing@@YA?AVAssertionResult@2@PEBD0_J1@Z)
[build] test_util.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class testing::AssertionResult __cdecl testing::internal::CmpHelperEQ(char const *,char const *,__int64,__int64)" (__imp_?CmpHelperEQ@internal@testing@@YA?AVAssertionResult@2@PEBD0_J1@Z)
[build] test_util.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl testing::internal2::PrintBytesInObjectTo(unsigned char const *,unsigned __int64,class std::basic_ostream<char,struct std::char_traits<char> > *)" (__imp_?PrintBytesInObjectTo@internal2@testing@@YAXPEBE_KPEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@@Z) referenced in function "public: static void __cdecl testing::internal2::TypeWithoutFormatter<class std::_Vector_iterator<class std::_Vector_val<struct std::_Simple_types<class arrow::compute::ExecNode *> > >,2>::PrintValue(class std::_Vector_iterator<class std::_Vector_val<struct std::_Simple_types<class arrow::compute::ExecNode *> > > const &,class std::basic_ostream<char,struct std::char_traits<char> > *)" (?PrintValue@?$TypeWithoutFormatter@V?$_Vector_iterator@V?$_Vector_val@U?$_Simple_types@PEAVExecNode@compute@arrow@@@std@@@std@@@std@@$01@internal2@testing@@SAXAEBV?$_Vector_iterator@V?$_Vector_val@U?$_Simple_types@PEAVExecNode@compute@arrow@@@std@@@std@@@std@@PEAV?$basic_ostream@DU?$char_traits@D@std@@@5@@Z)
[build] test_util.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) void __cdecl testing::internal2::PrintBytesInObjectTo(unsigned char const *,unsigned __int64,class std::basic_ostream<char,struct std::char_traits<char> > *)" (__imp_?PrintBytesInObjectTo@internal2@testing@@YAXPEBE_KPEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@@Z)
[build] debug\arrow_testing.dll : fatal error LNK1120: 2 unresolved externals
```

And then here's the config log:

```
[main] Configuring folder: arrow 
[proc] Executing command: "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.EXE" -DARROW_GANDIVA=OFF -DARROW_FILESYSTEM=ON -DARROW_S3=OFF -DARROW_HDFS=OFF -DARROW_PLASMA=OFF -DARROW_FLIGHT=OFF -DARROW_BUILD_INTEGRATION=ON -DARROW_BUILD_TESTS=ON -DARROW_EXTRA_ERROR_CONTEXT=ON -DCMAKE_BUILD_TYPE=Debug -DARROW_BUILD_STATIC=OFF -DARROW_PYTHON=ON -DARROW_ENGINE=ON -DARROW_MIMALLOC=ON -DARROW_PARQUET=ON -DARROW_WITH_BROTLI=ON -DARROW_WITH_BZ2=ON -DARROW_WITH_LZ4=ON -DARROW_WITH_RE2=ON -DARROW_WITH_SNAPPY=ON -DARROW_WITH_UTF8PROC=ON -DARROW_WITH_ZLIB=ON -DARROW_WITH_ZSTD=ON -DARROW_COMPUTE=ON -DARROW_CSV=ON -DARROW_DATASET=ON -DARROW_JSON=ON -DARROW_INSTALL_NAME_RPATH=OFF -DCMAKE_INSTALL_PREFIX=c:/Users/voltron/arrow/cpp/build/user-python-debug-windows/dist -DARROW_DEPENDENCY_SOURCE=VCPKG -DVCPKG_TARGET_TRIPLET=x64-windows -DA=x64 -Hc:/Users/voltron/arrow/cpp -Bc:/Users/voltron/arrow/cpp/build/user-python-debug-windows -G Ninja
[cmake] -- Building using CMake version: 3.21.21080301-MSVC_2
[cmake] -- Using vcpkg to find dependencies
[cmake] -- Using CMAKE_TOOLCHAIN_FILE: C:/Users/voltron/arrow/vcpkg/scripts/buildsystems/vcpkg.cmake
[cmake] -- Using VCPKG_ROOT: C:/Users/voltron/arrow/vcpkg
[cmake] -- Looking for installed packages in C:/Users/voltron/arrow/cpp/vcpkg_installed
[cmake] -- Using vcpkg installed libraries directory: C:/Users/voltron/arrow/cpp/vcpkg_installed
[cmake] -- Using VCPKG_TARGET_TRIPLET: x64-windows
[cmake] -- Using VCPKG_BUILD_TYPE: debug
[cmake] -- Using VCPKG_LIBRARY_LINKAGE: static
[cmake] -- Running vcpkg install
[cmake] Detecting compiler hash for triplet x64-windows...
[cmake] All requested packages are currently installed.
[cmake] Restored 0 packages from C:\Users\voltron\AppData\Local\vcpkg\archives in 145.7 us. Use --debug to see more details.
[cmake] 
[cmake] Total elapsed time: 3.479 s
[cmake] 
[cmake] The package abseil provides CMake targets:
[cmake] 
[cmake]     find_package(absl CONFIG REQUIRED)
[cmake]     # Note: 125 target(s) were omitted.
[cmake]     target_link_libraries(main PRIVATE absl::any absl::base absl::bits absl::city)
[cmake] 
[cmake] The package zlib is compatible with built-in CMake targets:
[cmake] 
[cmake]     find_package(ZLIB REQUIRED)
[cmake]     target_link_libraries(main PRIVATE ZLIB::ZLIB)
[cmake] 
[cmake] The package aws-sdk-cpp:x64-windows provides CMake targets:
[cmake] 
[cmake]     find_package(AWSSDK CONFIG COMPONENTS core dynamodb kinesis s3 REQUIRED)
[cmake]     target_include_directories(main PRIVATE ${AWSSDK_INCLUDE_DIRS})
[cmake]     target_link_libraries(main PRIVATE ${AWSSDK_LIBRARIES})
[cmake] 
[cmake] The package benchmark provides CMake targets:
[cmake] 
[cmake]     find_package(benchmark CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE benchmark::benchmark benchmark::benchmark_main)
[cmake] 
[cmake] The package boost is compatible with built-in CMake targets:
[cmake] 
[cmake]     find_package(Boost REQUIRED [COMPONENTS <libs>...])
[cmake]     target_link_libraries(main PRIVATE Boost::boost Boost::<lib1> Boost::<lib2> ...)
[cmake] 
[cmake] The package openssl is compatible with built-in CMake targets:
[cmake] 
[cmake]     find_package(OpenSSL REQUIRED)
[cmake]     target_link_libraries(main PRIVATE OpenSSL::SSL OpenSSL::Crypto)
[cmake] 
[cmake] The package brotli provides CMake targets:
[cmake] 
[cmake]     find_package(unofficial-brotli CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE unofficial::brotli::brotlidec unofficial::brotli::brotlienc unofficial::brotli::brotlicommon)
[cmake] 
[cmake] The package bzip2 is compatible with built-in CMake targets:
[cmake] 
[cmake]     find_package(BZip2 REQUIRED)
[cmake]     target_link_libraries(main PRIVATE BZip2::BZip2)
[cmake] 
[cmake] The package c-ares provides CMake targets:
[cmake] 
[cmake]     find_package(c-ares CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE c-ares::cares c-ares::cares_shared c-ares::cares_static)
[cmake] 
[cmake] The package curl provides CMake targets:
[cmake] 
[cmake]     find_package(CURL CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE CURL::libcurl)
[cmake] 
[cmake] The package flatbuffers provides CMake targets:
[cmake] 
[cmake]     find_package(Flatbuffers CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE flatbuffers::flatbuffers)
[cmake] 
[cmake] The package gflags provides CMake targets:
[cmake] 
[cmake]     find_package(gflags CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE gflags::gflags)
[cmake] 
[cmake] The package glog provides CMake targets:
[cmake] 
[cmake]     find_package(glog CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE glog::glog)
[cmake] 
[cmake] The package google-cloud-cpp provides CMake targets:
[cmake] 
[cmake]     find_package(google_cloud_cpp_common CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE google_cloud_cpp_common google-cloud-cpp::common)
[cmake] 
[cmake]     find_package(google_cloud_cpp_storage CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE CURL::libcurl google-cloud-cpp::storage google-cloud-cpp::experimental-storage-grpc)
[cmake] 
[cmake] The package re2 provides CMake targets:
[cmake] 
[cmake]     find_package(re2 CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE re2::re2)
[cmake] 
[cmake] The package protobuf provides CMake targets:
[cmake] 
[cmake]     find_package(protobuf CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE protobuf::libprotoc protobuf::libprotobuf protobuf::libprotobuf-lite)
[cmake] 
[cmake] The package grpc provides CMake targets:
[cmake] 
[cmake]     find_package(gRPC CONFIG REQUIRED)
[cmake]     # Note: 8 target(s) were omitted.
[cmake]     target_link_libraries(main PRIVATE gRPC::gpr gRPC::upb gRPC::grpc gRPC::grpc++)
[cmake] 
[cmake]     find_package(modules CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE re2::re2 c-ares::cares)
[cmake] 
[cmake] The package gtest provides CMake targets:
[cmake] 
[cmake]     find_package(GTest CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE GTest::gmock GTest::gtest GTest::gmock_main GTest::gtest_main)
[cmake] 
[cmake] The package lz4 provides CMake targets:
[cmake] 
[cmake]     find_package(lz4 CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE lz4::lz4)
[cmake] 
[cmake] The package zstd provides CMake targets:
[cmake] 
[cmake]     find_package(zstd CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE zstd::libzstd_shared)
[cmake] 
[cmake] The package snappy provides CMake targets:
[cmake] 
[cmake]     find_package(Snappy CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE Snappy::snappy)
[cmake] 
[cmake] The package orc does not provide CMake targets:
[cmake] 
[cmake]     find_package(protobuf CONFIG REQUIRED)
[cmake]     find_package(ZLIB REQUIRED)
[cmake]     find_package(Snappy CONFIG REQUIRED)
[cmake] 
[cmake]     find_library(LZ4_LIBRARY NAMES lz4d lz4)
[cmake]     find_library(ZSTD_LIBRARY NAMES zstd)
[cmake]     find_library(ORC_LIBRARY NAMES orc)
[cmake] 
[cmake]     target_link_libraries(main PRIVATE ${ORC_LIBRARY} protobuf::libprotoc protobuf::libprotobuf Snappy::snappy ${LZ4_LIBRARY} ${ZSTD_LIBRARY} ZLIB::ZLIB)
[cmake] 
[cmake] The package rapidjson provides CMake targets:
[cmake] 
[cmake]     find_package(RapidJSON CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE rapidjson)
[cmake] 
[cmake] The package sqlite3 provides CMake targets:
[cmake] 
[cmake]     find_package(unofficial-sqlite3 CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE unofficial::sqlite3::sqlite3)
[cmake] 
[cmake] The package thrift provides CMake targets:
[cmake] 
[cmake]     find_package(Thrift CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE thrift::thrift thriftz::thriftz thriftnb::thriftnb)
[cmake] 
[cmake] The package utf8proc provides CMake targets:
[cmake] 
[cmake]     find_package(unofficial-utf8proc CONFIG REQUIRED)
[cmake]     target_link_libraries(main PRIVATE utf8proc)
[cmake] 
[cmake] -- Running vcpkg install - done
[cmake] -- Arrow version: 7.0.0 (full: '7.0.0-SNAPSHOT')
[cmake] -- Arrow SO version: 700 (full: 700.0.0)
[cmake] -- clang-tidy 12 not found
[cmake] -- clang-format 12 not found
[cmake] -- Could NOT find ClangTools (missing: CLANG_FORMAT_BIN CLANG_TIDY_BIN) 
[cmake] -- infer not found
[cmake] -- Found Python3: C:/Users/voltron/AppData/Local/Programs/Python/Python310/python.exe (found version "3.10.1") found components: Interpreter 
[cmake] -- Found cpplint executable at C:/Users/voltron/arrow/cpp/build-support/cpplint.py
[cmake] -- System processor: AMD64
[cmake] -- Arrow build warning level: CHECKIN
[cmake] Configured for DEBUG build (set with cmake -DCMAKE_BUILD_TYPE={release,debug,...})
[cmake] -- Build Type: DEBUG
[cmake] -- Using SYSTEM approach to find dependencies
[cmake] -- ARROW_ABSL_BUILD_VERSION: 20210324.2
[cmake] -- ARROW_ABSL_BUILD_SHA256_CHECKSUM: 59b862f50e710277f8ede96f083a5bb8d7c9595376146838b9580be90374ee1f
[cmake] -- ARROW_AWSSDK_BUILD_VERSION: 1.8.133
[cmake] -- ARROW_AWSSDK_BUILD_SHA256_CHECKSUM: d6c495bc06be5e21dac716571305d77437e7cfd62a2226b8fe48d9ab5785a8d6
[cmake] -- ARROW_AWS_CHECKSUMS_BUILD_VERSION: v0.1.12
[cmake] -- ARROW_AWS_CHECKSUMS_BUILD_SHA256_CHECKSUM: 394723034b81cc7cd528401775bc7aca2b12c7471c92350c80a0e2fb9d2909fe
[cmake] -- ARROW_AWS_C_COMMON_BUILD_VERSION: v0.6.9
[cmake] -- ARROW_AWS_C_COMMON_BUILD_SHA256_CHECKSUM: 928a3e36f24d1ee46f9eec360ec5cebfe8b9b8994fe39d4fa74ff51aebb12717
[cmake] -- ARROW_AWS_C_EVENT_STREAM_BUILD_VERSION: v0.1.5
[cmake] -- ARROW_AWS_C_EVENT_STREAM_BUILD_SHA256_CHECKSUM: f1b423a487b5d6dca118bfc0d0c6cc596dc476b282258a3228e73a8f730422d4
[cmake] -- ARROW_BOOST_BUILD_VERSION: 1.75.0
[cmake] -- ARROW_BOOST_BUILD_SHA256_CHECKSUM: cb97b36e2295a321c34851e0455bc2630ad6c691d4f9f589170066cd11c835b4
[cmake] -- ARROW_BROTLI_BUILD_VERSION: v1.0.9
[cmake] -- ARROW_BROTLI_BUILD_SHA256_CHECKSUM: f9e8d81d0405ba66d181529af42a3354f838c939095ff99930da6aa9cdf6fe46
[cmake] -- ARROW_BZIP2_BUILD_VERSION: 1.0.8
[cmake] -- ARROW_BZIP2_BUILD_SHA256_CHECKSUM: ab5a03176ee106d3f0fa90e381da478ddae405918153cca248e682cd0c4a2269
[cmake] -- ARROW_CARES_BUILD_VERSION: 1.17.2
[cmake] -- ARROW_CARES_BUILD_SHA256_CHECKSUM: 4803c844ce20ce510ef0eb83f8ea41fa24ecaae9d280c468c582d2bb25b3913d
[cmake] -- ARROW_CRC32C_BUILD_VERSION: 1.1.2
[cmake] -- ARROW_CRC32C_BUILD_SHA256_CHECKSUM: ac07840513072b7fcebda6e821068aa04889018f24e10e46181068fb214d7e56
[cmake] -- ARROW_GBENCHMARK_BUILD_VERSION: v1.6.0
[cmake] -- ARROW_GBENCHMARK_BUILD_SHA256_CHECKSUM: 1f71c72ce08d2c1310011ea6436b31e39ccab8c2db94186d26657d41747c85d6
[cmake] -- ARROW_GFLAGS_BUILD_VERSION: v2.2.2
[cmake] -- ARROW_GFLAGS_BUILD_SHA256_CHECKSUM: 34af2f15cf7367513b352bdcd2493ab14ce43692d2dcd9dfc499492966c64dcf
[cmake] -- ARROW_GLOG_BUILD_VERSION: v0.5.0
[cmake] -- ARROW_GLOG_BUILD_SHA256_CHECKSUM: eede71f28371bf39aa69b45de23b329d37214016e2055269b3b5e7cfd40b59f5
[cmake] -- ARROW_GOOGLE_CLOUD_CPP_BUILD_VERSION: v1.31.1
[cmake] -- ARROW_GOOGLE_CLOUD_CPP_BUILD_SHA256_CHECKSUM: dc7cbf95b506a84b48cf71e0462985d262183edeaabdacaaee2109852394a609
[cmake] -- ARROW_GRPC_BUILD_VERSION: v1.35.0
[cmake] -- ARROW_GRPC_BUILD_SHA256_CHECKSUM: 27dd2fc5c9809ddcde8eb6fa1fa278a3486566dfc28335fca13eb8df8bd3b958
[cmake] -- ARROW_GTEST_BUILD_VERSION: 1.11.0
[cmake] -- ARROW_GTEST_BUILD_SHA256_CHECKSUM: b4870bf121ff7795ba20d20bcdd8627b8e088f2d1dab299a031c1034eddc93d5
[cmake] -- ARROW_JEMALLOC_BUILD_VERSION: 5.2.1
[cmake] -- ARROW_JEMALLOC_BUILD_SHA256_CHECKSUM: 34330e5ce276099e2e8950d9335db5a875689a4c6a56751ef3b1d8c537f887f6
[cmake] -- ARROW_LZ4_BUILD_VERSION: 8f61d8eb7c6979769a484cde8df61ff7c4c77765
[cmake] -- ARROW_LZ4_BUILD_SHA256_CHECKSUM: 5b072d848f2f93fddb97e1143e22c1bd7fa19fc8431ee69e21758190a88125b4
[cmake] -- ARROW_MIMALLOC_BUILD_VERSION: v1.7.3
[cmake] -- ARROW_MIMALLOC_BUILD_SHA256_CHECKSUM: 0f987bda01ca9df87ec90e9d98c63fa893ee61f3cca565e5ca5ed744fdcc5109
[cmake] -- ARROW_NLOHMANN_JSON_BUILD_VERSION: v3.10.2
[cmake] -- ARROW_NLOHMANN_JSON_BUILD_SHA256_CHECKSUM: 081ed0f9f89805c2d96335c3acfa993b39a0a5b4b4cef7edb68dd2210a13458c
[cmake] -- ARROW_OPENTELEMETRY_BUILD_VERSION: v1.1.0
[cmake] -- ARROW_OPENTELEMETRY_BUILD_SHA256_CHECKSUM: f8fd3adddd47be382dc79c19d7e7efcf86a0dfbb5a237db6e0618dbb7eb8e058
[cmake] -- ARROW_OPENTELEMETRY_PROTO_BUILD_VERSION: v0.11.0
[cmake] -- ARROW_OPENTELEMETRY_PROTO_BUILD_SHA256_CHECKSUM: 985367f8905e91018e636cbf0d83ab3f834b665c4f5899a27d10cae9657710e2
[cmake] -- ARROW_ORC_BUILD_VERSION: 1.7.2
[cmake] -- ARROW_ORC_BUILD_SHA256_CHECKSUM: ef39bae755116fecb07fb4334656f984ba8ce7cdb1c64de078d3ed186b286007
[cmake] -- ARROW_PROTOBUF_BUILD_VERSION: v3.18.1
[cmake] -- ARROW_PROTOBUF_BUILD_SHA256_CHECKSUM: b8ab9bbdf0c6968cf20060794bc61e231fae82aaf69d6e3577c154181991f576
[cmake] -- ARROW_RAPIDJSON_BUILD_VERSION: 1a803826f1197b5e30703afe4b9c0e7dd48074f5
[cmake] -- ARROW_RAPIDJSON_BUILD_SHA256_CHECKSUM: 0b6b780b6c534bfb0b23d29910bfe361e486bcfeaf106db8bc8995792072905a
[cmake] -- ARROW_RE2_BUILD_VERSION: 2021-11-01
[cmake] -- ARROW_RE2_BUILD_SHA256_CHECKSUM: 8c45f7fba029ab41f2a7e6545058d9eec94eef97ce70df58e92d85cfc08b4669
[cmake] -- ARROW_SNAPPY_BUILD_VERSION: 1.1.9
[cmake] -- ARROW_SNAPPY_BUILD_SHA256_CHECKSUM: 75c1fbb3d618dd3a0483bff0e26d0a92b495bbe5059c8b4f1c962b478b6e06e7
[cmake] -- ARROW_SNAPPY_OLD_BUILD_VERSION: 1.1.8
[cmake] -- ARROW_SNAPPY_OLD_BUILD_SHA256_CHECKSUM: 16b677f07832a612b0836178db7f374e414f94657c138e6993cbfc5dcc58651f
[cmake] -- ARROW_THRIFT_BUILD_VERSION: 0.13.0
[cmake] -- ARROW_THRIFT_BUILD_SHA256_CHECKSUM: 7ad348b88033af46ce49148097afe354d513c1fca7c607b59c33ebb6064b5179
[cmake] -- ARROW_UTF8PROC_BUILD_VERSION: v2.7.0
[cmake] -- ARROW_UTF8PROC_BUILD_SHA256_CHECKSUM: 4bb121e297293c0fd55f08f83afab6d35d48f0af4ecc07523ad8ec99aa2b12a1
[cmake] -- ARROW_XSIMD_BUILD_VERSION: aeec9c872c8b475dedd7781336710f2dd2666cb2
[cmake] -- ARROW_XSIMD_BUILD_SHA256_CHECKSUM: 0a841e6c8acf216150e4fc19fca8e29fbab9614b56ac7b96e56019264ca27b26
[cmake] -- ARROW_ZLIB_BUILD_VERSION: 1.2.11
[cmake] -- ARROW_ZLIB_BUILD_SHA256_CHECKSUM: c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
[cmake] -- ARROW_ZSTD_BUILD_VERSION: v1.5.1
[cmake] -- ARROW_ZSTD_BUILD_SHA256_CHECKSUM: dc05773342b28f11658604381afd22cb0a13e8ba17ff2bd7516df377060c18dd
[cmake] -- Boost include dir: C:/Users/voltron/arrow/cpp/vcpkg_installed/x64-windows/include
[cmake] -- Boost libraries: Boost::system;Boost::filesystem
[cmake] -- Providing CMake module for Snappy
[cmake] -- Providing CMake module for Brotli
[cmake] -- Building without OpenSSL support. Minimum OpenSSL version 1.0.2 required.
[cmake] -- Providing CMake module for Thrift
[cmake] -- Building (vendored) mimalloc from source
[cmake] -- Building gtest from source
[cmake] -- RapidJSON found. Headers: C:/Users/voltron/arrow/cpp/vcpkg_installed/x64-windows/share/rapidjson/../../include
[cmake] -- Building xsimd from source
[cmake] -- Providing CMake module for Lz4
[cmake] -- Using ZSTD_ROOT: C:/Users/voltron/arrow/cpp/vcpkg_installed/x64-windows
[cmake] -- Providing CMake module for zstd
[cmake] -- Providing CMake module for re2Alt
[cmake] -- Providing CMake module for utf8proc
[cmake] -- Found hdfs.h at: C:/Users/voltron/arrow/cpp/thirdparty/hadoop/include/hdfs.h
[cmake] -- All bundled static libraries: mimalloc::mimalloc
[cmake] -- CMAKE_C_FLAGS: /DWIN32 /D_WINDOWS /W3  /W3 /EHsc /wd5105 /bigobj /utf-8 /W3 /wd4365 /wd4267 /wd4838 /WX /wd4800 /wd4996 /wd4065  
[cmake] -- CMAKE_CXX_FLAGS:  /DWIN32 /D_WINDOWS  /GR /EHsc /D_SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING  /W3 /EHsc /wd5105 /bigobj /utf-8 /W3 /wd4365 /wd4267 /wd4838 /WX /wd4800 /wd4996 /wd4065  
[cmake] -- Could NOT find Backtrace (missing: Backtrace_LIBRARY Backtrace_INCLUDE_DIR) 
[cmake] -- Found Python3: C:/Users/voltron/AppData/Local/Programs/Python/Python310/python.exe (found version "3.10.1") found components: Interpreter Development NumPy Development.Module Development.Embed 
[cmake] -- ---------------------------------------------------------------------
[cmake] -- Arrow version:                                 7.0.0-SNAPSHOT
[cmake] -- 
[cmake] -- Build configuration summary:
[cmake] --   Generator: Ninja
[cmake] --   Build type: DEBUG
[cmake] --   Source directory: C:/Users/voltron/arrow/cpp
[cmake] --   Install prefix: C:/Users/voltron/arrow/cpp/build/user-python-debug-windows/dist
[cmake] -- 
[cmake] -- Compile and link options:
[cmake] -- 
[cmake] --   ARROW_CXXFLAGS="" [default=""]
[cmake] --       Compiler flags to append when compiling Arrow
[cmake] --   ARROW_BUILD_STATIC=OFF [default=ON]
[cmake] --       Build static libraries
[cmake] --   ARROW_BUILD_SHARED=ON [default=ON]
[cmake] --       Build shared libraries
[cmake] --   ARROW_PACKAGE_KIND="" [default=""]
[cmake] --       Arbitrary string that identifies the kind of package
[cmake] --       (for informational purposes)
[cmake] --   ARROW_GIT_ID=f585a470539d61cbc237b66a1851149d28adc176 [default=""]
[cmake] --       The Arrow git commit id (if any)
[cmake] --   ARROW_GIT_DESCRIPTION=apache-arrow-7.0.0.dev-505-gf585a4705 [default=""]
[cmake] --       The Arrow git commit description (if any)
[cmake] --   ARROW_NO_DEPRECATED_API=OFF [default=OFF]
[cmake] --       Exclude deprecated APIs from build
[cmake] --   ARROW_USE_CCACHE=ON [default=ON]
[cmake] --       Use ccache when compiling (if available)
[cmake] --   ARROW_USE_LD_GOLD=OFF [default=OFF]
[cmake] --       Use ld.gold for linking on Linux (if available)
[cmake] --   ARROW_USE_PRECOMPILED_HEADERS=OFF [default=OFF]
[cmake] --       Use precompiled headers when compiling
[cmake] --   ARROW_SIMD_LEVEL=SSE4_2 [default=NONE|SSE4_2|AVX2|AVX512|NEON|DEFAULT]
[cmake] --       Compile-time SIMD optimization level
[cmake] --   ARROW_RUNTIME_SIMD_LEVEL=MAX [default=NONE|SSE4_2|AVX2|AVX512|MAX]
[cmake] --       Max runtime SIMD optimization level
[cmake] --   ARROW_ARMV8_ARCH=armv8-a [default=armv8-a|armv8-a+crc+crypto]
[cmake] --       Arm64 arch and extensions
[cmake] --   ARROW_ALTIVEC=ON [default=ON]
[cmake] --       Build with Altivec if compiler has support
[cmake] --   ARROW_RPATH_ORIGIN=OFF [default=OFF]
[cmake] --       Build Arrow libraries with RATH set to $ORIGIN
[cmake] --   ARROW_INSTALL_NAME_RPATH=OFF [default=ON]
[cmake] --       Build Arrow libraries with install_name set to @rpath
[cmake] --   ARROW_GGDB_DEBUG=ON [default=ON]
[cmake] --       Pass -ggdb flag to debug builds
[cmake] -- 
[cmake] -- Test and benchmark options:
[cmake] -- 
[cmake] --   ARROW_BUILD_EXAMPLES=OFF [default=OFF]
[cmake] --       Build the Arrow examples
[cmake] --   ARROW_BUILD_TESTS=ON [default=OFF]
[cmake] --       Build the Arrow googletest unit tests
[cmake] --   ARROW_ENABLE_TIMING_TESTS=ON [default=ON]
[cmake] --       Enable timing-sensitive tests
[cmake] --   ARROW_BUILD_INTEGRATION=ON [default=OFF]
[cmake] --       Build the Arrow integration test executables
[cmake] --   ARROW_BUILD_BENCHMARKS=OFF [default=OFF]
[cmake] --       Build the Arrow micro benchmarks
[cmake] --   ARROW_BUILD_BENCHMARKS_REFERENCE=OFF [default=OFF]
[cmake] --       Build the Arrow micro reference benchmarks
[cmake] --   ARROW_BUILD_OPENMP_BENCHMARKS=OFF [default=OFF]
[cmake] --       Build the Arrow benchmarks that rely on OpenMP
[cmake] --   ARROW_BUILD_DETAILED_BENCHMARKS=OFF [default=OFF]
[cmake] --       Build benchmarks that do a longer exploration of performance
[cmake] --   ARROW_TEST_LINKAGE=shared [default=shared|static]
[cmake] --       Linkage of Arrow libraries with unit tests executables.
[cmake] --   ARROW_FUZZING=OFF [default=OFF]
[cmake] --       Build Arrow Fuzzing executables
[cmake] --   ARROW_LARGE_MEMORY_TESTS=OFF [default=OFF]
[cmake] --       Enable unit tests which use large memory
[cmake] -- 
[cmake] -- Lint options:
[cmake] -- 
[cmake] --   ARROW_ONLY_LINT=OFF [default=OFF]
[cmake] --       Only define the lint and check-format targets
[cmake] --   ARROW_VERBOSE_LINT=OFF [default=OFF]
[cmake] --       If off, 'quiet' flags will be passed to linting tools
[cmake] --   ARROW_GENERATE_COVERAGE=OFF [default=OFF]
[cmake] --       Build with C++ code coverage enabled
[cmake] -- 
[cmake] -- Checks options:
[cmake] -- 
[cmake] --   ARROW_TEST_MEMCHECK=OFF [default=OFF]
[cmake] --       Run the test suite using valgrind --tool=memcheck
[cmake] --   ARROW_USE_ASAN=OFF [default=OFF]
[cmake] --       Enable Address Sanitizer checks
[cmake] --   ARROW_USE_TSAN=OFF [default=OFF]
[cmake] --       Enable Thread Sanitizer checks
[cmake] --   ARROW_USE_UBSAN=OFF [default=OFF]
[cmake] --       Enable Undefined Behavior sanitizer checks
[cmake] -- 
[cmake] -- Project component options:
[cmake] -- 
[cmake] --   ARROW_BUILD_UTILITIES=OFF [default=OFF]
[cmake] --       Build Arrow commandline utilities
[cmake] --   ARROW_COMPUTE=ON [default=OFF]
[cmake] --       Build the Arrow Compute Modules
[cmake] --   ARROW_CSV=ON [default=OFF]
[cmake] --       Build the Arrow CSV Parser Module
[cmake] --   ARROW_CUDA=OFF [default=OFF]
[cmake] --       Build the Arrow CUDA extensions (requires CUDA toolkit)
[cmake] --   ARROW_DATASET=ON [default=OFF]
[cmake] --       Build the Arrow Dataset Modules
[cmake] --   ARROW_ENGINE=ON [default=OFF]
[cmake] --       Build the Arrow Execution Engine
[cmake] --   ARROW_FILESYSTEM=ON [default=OFF]
[cmake] --       Build the Arrow Filesystem Layer
[cmake] --   ARROW_FLIGHT=OFF [default=OFF]
[cmake] --       Build the Arrow Flight RPC System (requires GRPC, Protocol Buffers)
[cmake] --   ARROW_FLIGHT_SQL=OFF [default=OFF]
[cmake] --       Build the Arrow Flight SQL extension
[cmake] --   ARROW_GANDIVA=OFF [default=OFF]
[cmake] --       Build the Gandiva libraries
[cmake] --   ARROW_GCS=OFF [default=OFF]
[cmake] --       Build Arrow with GCS support (requires the GCloud SDK for C++)
[cmake] --   ARROW_HDFS=ON [default=OFF]
[cmake] --       Build the Arrow HDFS bridge
[cmake] --   ARROW_HIVESERVER2=OFF [default=OFF]
[cmake] --       Build the HiveServer2 client and Arrow adapter
[cmake] --   ARROW_IPC=ON [default=ON]
[cmake] --       Build the Arrow IPC extensions
[cmake] --   ARROW_JEMALLOC=OFF [default=OFF]
[cmake] --       Build the Arrow jemalloc-based allocator
[cmake] --   ARROW_JNI=OFF [default=OFF]
[cmake] --       Build the Arrow JNI lib
[cmake] --   ARROW_JSON=ON [default=OFF]
[cmake] --       Build Arrow with JSON support (requires RapidJSON)
[cmake] --   ARROW_MIMALLOC=ON [default=OFF]
[cmake] --       Build the Arrow mimalloc-based allocator
[cmake] --   ARROW_PARQUET=ON [default=OFF]
[cmake] --       Build the Parquet libraries
[cmake] --   ARROW_ORC=OFF [default=OFF]
[cmake] --       Build the Arrow ORC adapter
[cmake] --   ARROW_PLASMA=OFF [default=OFF]
[cmake] --       Build the plasma object store along with Arrow
[cmake] --   ARROW_PLASMA_JAVA_CLIENT=OFF [default=OFF]
[cmake] --       Build the plasma object store java client
[cmake] --   ARROW_PYTHON=ON [default=OFF]
[cmake] --       Build the Arrow CPython extensions
[cmake] --   ARROW_S3=OFF [default=OFF]
[cmake] --       Build Arrow with S3 support (requires the AWS SDK for C++)
[cmake] --   ARROW_SKYHOOK=OFF [default=OFF]
[cmake] --       Build the Skyhook libraries
[cmake] --   ARROW_TENSORFLOW=OFF [default=OFF]
[cmake] --       Build Arrow with TensorFlow support enabled
[cmake] --   ARROW_TESTING=ON [default=OFF]
[cmake] --       Build the Arrow testing libraries
[cmake] -- 
[cmake] -- Thirdparty toolchain options:
[cmake] -- 
[cmake] --   ARROW_DEPENDENCY_SOURCE=SYSTEM [default=AUTO|BUNDLED|SYSTEM|CONDA|VCPKG|BREW]
[cmake] --       Method to use for acquiring arrow's build dependencies
[cmake] --   ARROW_VERBOSE_THIRDPARTY_BUILD=OFF [default=OFF]
[cmake] --       Show output from ExternalProjects rather than just logging to files
[cmake] --   ARROW_DEPENDENCY_USE_SHARED=ON [default=ON]
[cmake] --       Link to shared libraries
[cmake] --   ARROW_BOOST_USE_SHARED=ON [default=ON]
[cmake] --       Rely on boost shared libraries where relevant
[cmake] --   ARROW_BROTLI_USE_SHARED=ON [default=ON]
[cmake] --       Rely on Brotli shared libraries where relevant
[cmake] --   ARROW_BZ2_USE_SHARED=ON [default=ON]
[cmake] --       Rely on Bz2 shared libraries where relevant
[cmake] --   ARROW_GFLAGS_USE_SHARED=ON [default=ON]
[cmake] --       Rely on GFlags shared libraries where relevant
[cmake] --   ARROW_GRPC_USE_SHARED=ON [default=ON]
[cmake] --       Rely on gRPC shared libraries where relevant
[cmake] --   ARROW_LZ4_USE_SHARED=ON [default=ON]
[cmake] --       Rely on lz4 shared libraries where relevant
[cmake] --   ARROW_OPENSSL_USE_SHARED=ON [default=ON]
[cmake] --       Rely on OpenSSL shared libraries where relevant
[cmake] --   ARROW_PROTOBUF_USE_SHARED=ON [default=ON]
[cmake] --       Rely on Protocol Buffers shared libraries where relevant
[cmake] --   ARROW_THRIFT_USE_SHARED=OFF [default=OFF]
[cmake] --       Rely on thrift shared libraries where relevant
[cmake] --   ARROW_UTF8PROC_USE_SHARED=ON [default=ON]
[cmake] --       Rely on utf8proc shared libraries where relevant
[cmake] --   ARROW_SNAPPY_USE_SHARED=ON [default=ON]
[cmake] --       Rely on snappy shared libraries where relevant
[cmake] --   ARROW_UTF8PROC_USE_SHARED=ON [default=ON]
[cmake] --       Rely on utf8proc shared libraries where relevant
[cmake] --   ARROW_ZSTD_USE_SHARED=ON [default=ON]
[cmake] --       Rely on zstd shared libraries where relevant
[cmake] --   ARROW_USE_GLOG=OFF [default=OFF]
[cmake] --       Build libraries with glog support for pluggable logging
[cmake] --   ARROW_WITH_BACKTRACE=ON [default=ON]
[cmake] --       Build with backtrace support
[cmake] --   ARROW_WITH_OPENTELEMETRY=OFF [default=OFF]
[cmake] --       Build libraries with OpenTelemetry support for distributed tracing
[cmake] --   ARROW_WITH_BROTLI=ON [default=OFF]
[cmake] --       Build with Brotli compression
[cmake] --   ARROW_WITH_BZ2=ON [default=OFF]
[cmake] --       Build with BZ2 compression
[cmake] --   ARROW_WITH_LZ4=ON [default=OFF]
[cmake] --       Build with lz4 compression
[cmake] --   ARROW_WITH_SNAPPY=ON [default=OFF]
[cmake] --       Build with Snappy compression
[cmake] --   ARROW_WITH_ZLIB=ON [default=OFF]
[cmake] --       Build with zlib compression
[cmake] --   ARROW_WITH_ZSTD=ON [default=OFF]
[cmake] --       Build with zstd compression
[cmake] --   ARROW_WITH_UTF8PROC=ON [default=ON]
[cmake] --       Build with support for Unicode properties using the utf8proc library
[cmake] --       (only used if ARROW_COMPUTE is ON or ARROW_GANDIVA is ON)
[cmake] --   ARROW_WITH_RE2=ON [default=ON]
[cmake] --       Build with support for regular expressions using the re2 library
[cmake] --       (only used if ARROW_COMPUTE or ARROW_GANDIVA is ON)
[cmake] -- 
[cmake] -- MSVC options:
[cmake] -- 
[cmake] --   MSVC_LINK_VERBOSE=OFF [default=OFF]
[cmake] --       Pass verbose linking options when linking libraries and executables
[cmake] --   BROTLI_MSVC_STATIC_LIB_SUFFIX=-static [default="-static"]
[cmake] --       Brotli static lib suffix used on Windows with MSVC
[cmake] --   PROTOBUF_MSVC_STATIC_LIB_SUFFIX="" [default=""]
[cmake] --       Protobuf static lib suffix used on Windows with MSVC
[cmake] --   RE2_MSVC_STATIC_LIB_SUFFIX=_static [default="_static"]
[cmake] --       re2 static lib suffix used on Windows with MSVC
[cmake] --   SNAPPY_MSVC_STATIC_LIB_SUFFIX="" [default=""]
[cmake] --       Snappy static lib suffix used on Windows with MSVC
[cmake] --   LZ4_MSVC_STATIC_LIB_SUFFIX="" [default="_static"]
[cmake] --       Lz4 static lib suffix used on Windows with MSVC
[cmake] --   ZSTD_MSVC_STATIC_LIB_SUFFIX=_static [default="_static"]
[cmake] --       ZStd static lib suffix used on Windows with MSVC
[cmake] --   ARROW_USE_STATIC_CRT=OFF [default=OFF]
[cmake] --       Build Arrow with statically linked CRT
[cmake] -- 
[cmake] -- Parquet options:
[cmake] -- 
[cmake] --   PARQUET_MINIMAL_DEPENDENCY=OFF [default=OFF]
[cmake] --       Depend only on Thirdparty headers to build libparquet.
[cmake] --       Always OFF if building binaries
[cmake] --   PARQUET_BUILD_EXECUTABLES=OFF [default=OFF]
[cmake] --       Build the Parquet executable CLI tools. Requires static libraries to be built.
[cmake] --   PARQUET_BUILD_EXAMPLES=OFF [default=OFF]
[cmake] --       Build the Parquet examples. Requires static libraries to be built.
[cmake] --   PARQUET_REQUIRE_ENCRYPTION=OFF [default=OFF]
[cmake] --       Build support for encryption. Fail if OpenSSL is not found
[cmake] -- 
[cmake] -- Gandiva options:
[cmake] -- 
[cmake] --   ARROW_GANDIVA_JAVA=OFF [default=OFF]
[cmake] --       Build the Gandiva JNI wrappers
[cmake] --   ARROW_GANDIVA_STATIC_LIBSTDCPP=OFF [default=OFF]
[cmake] --       Include -static-libstdc++ -static-libgcc when linking with
[cmake] --       Gandiva static libraries
[cmake] --   ARROW_GANDIVA_PC_CXX_FLAGS="" [default=""]
[cmake] --       Compiler flags to append when pre-compiling Gandiva operations
[cmake] -- 
[cmake] -- Advanced developer options:
[cmake] -- 
[cmake] --   ARROW_EXTRA_ERROR_CONTEXT=ON [default=OFF]
[cmake] --       Compile with extra error context (line numbers, code)
[cmake] --   ARROW_OPTIONAL_INSTALL=OFF [default=OFF]
[cmake] --       If enabled install ONLY targets that have already been built. Please be
[cmake] --       advised that if this is enabled 'install' will fail silently on components
[cmake] --       that have not been built
[cmake] --   Outputting build configuration summary to C:/Users/voltron/arrow/cpp/build/user-python-debug-windows/cmake_summary.json
[cmake] -- Configuring done
[cmake] -- Generating done
[cmake] -- Build files have been written to: C:/Users/voltron/arrow/cpp/build/user-python-debug-windows
[cpptools] The build configurations generated do not contain the active build configuration. Using 'DEBUG' for CMAKE_BUILD_TYPE instead of 'Debug' to ensure that IntelliSense configurations can be found
```